src/rascal/structure_managers/property_block_sparse.hh

```diff
@@ -969,6 +970,12 @@ namespace rascal {
       return arr;
     }
 
+    double sum() const { return this->values.sum(); }
+
+    double l1_norm() const {
```

Comment:
in Eigen most reductions like `sum` return the value so I am following their example. Also we use `pairs()` instead of `get_pairs()` in our code so it seems consistent. What do you think ?